runtime/onert/backend/cpu/ops/GatherLayer.cc

```diff
@@ -83,7 +83,7 @@ void GatherLayer::runByGGMLQuantInputType()
   if (getShape(_input).DimensionsCount() != 2)
     throw std::runtime_error("Gather: block quantized input tensor must be rank 2");
 
-  if (getShape(_indices).DimensionsCount() > 4 &&
+  if (getShape(_indices).DimensionsCount() >= 4 &&
```

Comment:
A few days ago, I also thought it was wrong at a glance. However, I think it is true.

It is the negation of the condition in comment.

>  // Indice: rank < 4 or rank 4 with dim(0) = 1, INT32

not (A or B) = (not A) and (not B)